src/k8s-engine/client.ts

```diff
@@ -217,7 +217,7 @@ export class KubeClient extends events.EventEmitter {
    * @param {string} endpointName the name of an endpoint that controls ready pods.
    * @returns {Promise<k8s.V1Pod?>}
    */
-  protected async getActivePod(namespace: string, endpointName: string): Promise<k8s.V1Pod | null> {
+  async getActivePod(namespace: string, endpointName: string): Promise<k8s.V1Pod | null> {
```

Comment:
If we're making this available, we should also add a JSDoc block.

Reply:
It has one.

Reply:
Oh, right, it does. Sorry about that.